src/System.Net.Primitives/src/System/Net/CookieCollection.cs

```diff
@@ -24,7 +24,8 @@ internal enum Stamp
 
         private readonly ArrayList m_list = new ArrayList();
 
-        private DateTime m_timeStamp = DateTime.MinValue; // Do not rename (binary serialization)
+        private int m_version; // Do not rename (binary serialization)
+        private DateTime m_TimeStamp = DateTime.MinValue; // Do not rename (binary serialization)
         private bool m_has_other_versions; // Do not rename (binary serialization)
```

Comment:
The question is what we do when a payload arrives at Core that DOES contain m_IsReadOnly (which we don't have a field for). Does the deserializer just ignore the unexpected extra value? If we are currently testing with a 4.6.1 blob that does contain this value, then it's clearly working and we're fine.